src/pkg/utils/utils.go

```diff
@@ -500,9 +500,12 @@ func HumanDuration(duration int64) string {
 }
 
 // ImageReferenceCanBeID checks if 'image' might be the ID of an image
-func ImageReferenceCanBeID(image string) (bool, error) {
-	matched, err := regexp.MatchString("^[a-f0-9]\\{6,64\\}$", image)
-	return matched, err
+func ImageReferenceCanBeID(image string) bool {
+	matched, err := regexp.MatchString("^[a-f0-9]{6,64}$", image)
```

Comment:
Ah, I missed the change in the escaping.

This is potentially an actual behavioral change, so we should put it in a separate commit. Especially because regular expressions are really weird. eg., `grep` differentiates between so-called basic and extended regular expressions where the characters `?`, `+`, `{`, `|`, `(` and `)` need or need not be escaped.

Reply:
I moved the change to a separate commit.